### PR TITLE
Adding first unit-test

### PR DESCRIPTION
**Pull request by AdamSaleh**
Added a unit test, that can be found in file [tests/library/eachComponentTestSpec.groovy](https://github.com/feedhenry/fh-pipeline-library/pull/39/files#diff-808fb3b3020e80942ca3e83870262129)

Rest of the configuration has been taken from https://github.com/macg33zr/pipelineUnit

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1 @@
+language: java
```
Comment (grdryn) [inline .travis.yml:1]: Travis?! 🤔

**Review by grdryn — APPROVED** [review]
@AdamSaleh I don't understand a lot of this. I'm not sure if that's a problem? Is the stuff in `./tests/support` copied from somewhere?

File: .travis.yml (hunk 1 above)
Comment (grdryn) [reply .travis.yml:1]: If you want to keep this in, is it ok if I remove it when adding the Jenkinsfile?

Comment (AdamSaleh) [reply .travis.yml:1]: Yep :)

**Comment by grdryn** [comment]
@AdamSaleh After running this locally, should `.gradle/` and `build` be added to a `.gitignore` file?

**Comment by grdryn** [comment]
@AdamSaleh master now has a Jenkinsfile, so I've enforced that check on all PRs. Once you rebase this PR, you should get that Jenkinsfile and you'll be able to trigger that job from Jenkins.

**Comment by grdryn** [comment]
Closing as out-of-date. Thanks!

**Pull request closed without merging**